File: Project-Backend/src/main/java/com/qa/model/PlayerCard.java
package com.qa.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class PlayerCard {
	@Id
	@GeneratedValue
	private int id;
	private String name;
	private String gamerTag;
	private String character;
	
	public int getId() {
		return id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getGamerTag() {
		return gamerTag;
	}
	public void setGamerTag(String gamerTag) {
		this.gamerTag = gamerTag;
	}
	public String getCharacter() {
		return character;
	}
	public void setCharacter(String character) {
		this.character = character;
	}
	
	
}

File: Playercard-fullstack-app/Project-Backend/src/main/webapp/playercard.js


let userName;
let tag;
let char;

let profile = {
    name: userName,
    gamerTag: tag,
    character: char
}

const setName = (name) => { userName = name.value; };
const setGamerTag = (gamerTag) => { tag = gamerTag.value; };
const setCharacter = (character) => { char = character.value; };

function submit() {
    profile.name = userName;
    profile.gamerTag = tag;
    profile.character = char;
}

function newPlayer() {
    submit();
    userData = JSON.stringify(profile);
    fetchData("playercard", 'POST', userData)
        .then((x) => {
            localStorage.setItem("playercard", x);
            window.location = "card.html";
        }).catch((reason) => {
            console.log(reason)
        });
}

function goHome() {
    window.location = "home.html";
}
